ethr-status/src/main/java/me/uport/sdk/ethr_status/EthrStatusResolver.kt

```diff
@@ -47,22 +52,52 @@ class EthrStatusResolver : StatusResolver {
     private suspend fun runCredentialCheck(
         credential: String,
         status: StatusEntry,
-        issuer: String
+        didDoc: DIDDocument
     ): EthrStatus {
         val (registryAddress, network) = parseRegistryId(status.id)
 
         val ethNetwork = Networks.get(network)
         val rpc = JsonRPC(ethNetwork.rpcUrl)
         val credentialHash = credential.toByteArray().keccak()
 
-        val encodedMethodCall = Revocation.Revoked.encode(
-            Solidity.Address(extractAddress(issuer).hexToBigInteger()),
-            Solidity.Bytes32(credentialHash)
-        )
+        val revokers = getValidRevokers(didDoc)
+
+        revokers.forEach { revoker ->
+            val encodedMethodCall = Revocation.Revoked.encode(
+                Solidity.Address(revoker.hexToBigInteger()),
+                Solidity.Bytes32(credentialHash)
+            )
+
+            val result = rpc.ethCall(registryAddress, encodedMethodCall)
+            if (result.hexToBigInteger() > BigInteger.ZERO) {
+                return EthrStatus(BigInteger.ONE)
```

Comment:
should return the lowest non-zero block number here instead of `BigInteger.ONE`

or `BigInteger.ZERO` if all the results are ZERO